pkg/server/batch_check_test.go

```diff
@@ -0,0 +1,224 @@
+package server
+
+import (
+	"context"
+	"errors"
+	"testing"
+
+	"go.uber.org/goleak"
+
+	"github.com/openfga/openfga/cmd/util"
+	"github.com/openfga/openfga/pkg/testutils"
+	"github.com/openfga/openfga/pkg/tuple"
+
+	openfgav1 "github.com/openfga/api/proto/openfga/v1"
+	"github.com/stretchr/testify/require"
+
+	"github.com/openfga/openfga/internal/condition"
+	"github.com/openfga/openfga/internal/graph"
+	"github.com/openfga/openfga/pkg/server/commands"
+)
+
+func TestBatchCheckUsesTypesystemModel(t *testing.T) {
+	t.Cleanup(func() {
+		goleak.VerifyNone(t)
+	})
```

Comment:
I haven't cross-checked every test here, but as a reminder we need to have this for all appropriate tests.